src/screens/ProjectDetail/ProjectScript.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * @function isCloudinaryVideo
+ * @author Raymond Ndibe <ndiberaymond1@gmail.com>
+ *
+ * @todo - describe function's signature
+ */
+export const isCloudinaryVideo = (url) =>
+  url.search('cloudinary.com') > -1 ? true : false;
```

Comment:
can be simplified to 
```suggestion
  url.includes('cloudinary.com');
```
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes